src/group/organizationGroup/organizationGroup.controller.ts

```diff
@@ -6,10 +6,22 @@ import { PersonRepository } from '../../person/person.repository';
 import * as _ from 'lodash';
 import { sortObjectsByIDArray, promiseAllWithFails, asyncForEach } from '../../utils';
 import { ValidationError, ResourceNotFoundError } from '../../types/error';
+import { OrganizationGroupTextSearch } from './organizationGroup.textSearch';
+import organizationGroupElasticRepository from './organizationGroup.elasticSearchRepository';
+
+type GroupFilters = {
+  underGroupId: string
+};
+
+export type GroupQuery = {
+  name: string,
+  hierarchy: string;
+};
 
 export class OrganizationGroup {
   static _organizationGroupRepository: OrganizationGroupRepository = new OrganizationGroupRepository();
   static _personRepository: PersonRepository = new PersonRepository();
+  static _organizationGroupTextSearch: OrganizationGroupTextSearch = organizationGroupElasticRepository;
```

Comment:
Is this a public or a private member variable? 🤔 
An underscore usually indicates a private member variable, but in typescript all members of a class are public by default.